versions.tf

```diff
@@ -3,9 +3,12 @@
 # ---------------------------------------------------------------------------------------------------------------------
 
 terraform {
-  required_version = ">= 0.12.20, < 0.15"
+  required_version = ">= 0.12.20, < 0.16"
 
   required_providers {
-    github = ">= 2.9, < 4.0, != 3.1.0"
+    github = {
+      source  = "integrations/github"
+      version = ">= 2.9, < 4.0, != 3.1.0"
+    }
```

Comment:
this will break compatibility as this a 0.13+ feature?

Reply:
we should drop compatibility to v0.13 this. We need to change to `integrations/github` since github took over the development of the provider and the old one is deprecated

Reply:
wdyt @mariux ?

Reply:
I'd argue we drop compatibility to v0.13 then since the GitHub provider has been moved to GitHub and is not managed by hashicorp anymore

Reply:
actually modules should never define source of providers... this is userland decision.